test/test_main.py

```diff
@@ -86,6 +86,12 @@ def test_get_version_warning(
     assert len(msg.split("\n")) == outlen
 
 
+def test_get_version_warning_no_pip(mocker: MockerFixture) -> None:
+    """Test that we do not display any message is install method is not pip."""
```

Comment:
Minor typo in the docstring: """Test that we do not display any message **_if_** install method is not pip."""